lib/Doctrine/ORM/Tools/Export/Driver/PhpExporter.php

```diff
@@ -82,6 +82,16 @@ public function exportClassMetadata(ClassMetadataInfo $metadata)
             }
         }
 
+        if (0 !== count($metadata->entityListeners)) {
+            foreach ($metadata->entityListeners as $event => $entityListenerConfig) {
+                foreach ($entityListenerConfig as $entityListener) {
+                    $class = $entityListener['class'];
+                    $method = $entityListener['method'];
+                    $lines[] = "\$metadata->addEntityListener('$event', '$class', '$method');";
```

Comment:
Please don't interpolate. Use `sprintf()` instead. Also, please do use `var_export()` on values that end up being in PHP code.